### DISCUSSION TITLE
How can I use logging in Python and have it registered in AWS CloudWatch

### DISCUSSION THREAD

**Discussion by gabrielmpaula** [awslabs/aws-lambda-web-adapter#544] [Q&A]
Hello, I have been using Lambda Adapter to create a Lambda Function with streaming response, however I haven't been able to successfully configure the logging module in Python and print it in AWS CloudWatch. Do you know how can I achieve that?

**Comment by bnusunny** [comment]
Jus send the logs to the console (stdout/stderr). Lambda will collect them and send to CloudWatch Logs automatically.
